reframe/utility/__init__.py

```diff
@@ -24,6 +25,23 @@
 from . import typecheck as typ
 
 
+def get_hostname_cmd(hostname_cmd, logger=None):
+    if hostname_cmd == 'xthostname':
```

Comment:
This is a tiny thing, but maybe `xthostnamex` is a better name than `xthostname`, since it is based on the name of the cray file?

Reply:
no, sorry, I was checking the behavior in case the file doesn't exist. I pushed it by mistake with the misspelled name.